user_guide_src/source/libraries/incomingrequest.rst

```diff
@@ -133,6 +133,14 @@ arrays, pass in ``true`` as the first parameter.
 The second and third parameters match up to the ``depth`` and ``options`` arguments of the
 `json_decode <http://php.net/manual/en/function.json-decode.php>`_ PHP function.
 
+**Getting Raw data (PUT, PATCH, DELETE)**
+
+Finally, you can grab the contents of php://input as a raw stream with ``getRawInput()``.
+
+	$data = $request->getRawInput();
+
+By default, this will convert param string to an array. 
```

Comment:
"By default" implies it's possible to get it back in a different format, but currently we can only retrieve it as an array.

If you could show a small example of what they should expect to get, that would be awesome. I need to add that to the getJSON docs also, it looks like.